staging/nvidia/charts/nvidia-driver/templates/configmap.yaml

```diff
@@ -0,0 +1,274 @@
+    enabled=1
+
+    [C7.4.1708-centosplus]
+    name=CentOS-7.4.1708 - CentOSPlus
+    baseurl=http://vault.centos.org/7.4.1708/centosplus/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.4.1708-fasttrack]
+    name=CentOS-7.4.1708 - Fasttrack
+    baseurl=http://vault.centos.org/7.4.1708/fasttrack/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    # C7.5.1804
+    [C7.5.1804-base]
+    name=CentOS-7.5.1804 - Base
+    baseurl=http://vault.centos.org/7.5.1804/os/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.5.1804-updates]
+    name=CentOS-7.5.1804 - Updates
+    baseurl=http://vault.centos.org/7.5.1804/updates/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.5.1804-extras]
+    name=CentOS-7.5.1804 - Extras
+    baseurl=http://vault.centos.org/7.5.1804/extras/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.5.1804-centosplus]
+    name=CentOS-7.5.1804 - CentOSPlus
+    baseurl=http://vault.centos.org/7.5.1804/centosplus/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.5.1804-fasttrack]
+    name=CentOS-7.5.1804 - Fasttrack
+    baseurl=http://vault.centos.org/7.5.1804/fasttrack/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    # C7.6.1810
+    [C7.6.1810-base]
+    name=CentOS-7.6.1810 - Base
+    baseurl=http://vault.centos.org/7.6.1810/os/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.6.1810-updates]
+    name=CentOS-7.6.1810 - Updates
+    baseurl=http://vault.centos.org/7.6.1810/updates/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.6.1810-extras]
+    name=CentOS-7.6.1810 - Extras
+    baseurl=http://vault.centos.org/7.6.1810/extras/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.6.1810-centosplus]
+    name=CentOS-7.6.1810 - CentOSPlus
+    baseurl=http://vault.centos.org/7.6.1810/centosplus/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+
+    [C7.6.1810-fasttrack]
+    name=CentOS-7.6.1810 - Fasttrack
+    baseurl=http://vault.centos.org/7.6.1810/fasttrack/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+    enabled=1
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "nvidia-driver.fullname" . }}-centos7-vault-repo
+  namespace: kube-system
+data:
+  wrapper.sh: |
+    #!/bin/bash
+    mkdir -p /lib/modules/$(uname -r)/proc
+    cp /proc/version /lib/modules/proc/version
+    /usr/local/bin/nvidia-driver init -a
```

Comment:
Just curious, is this command idempotent? Thinking of an upgrade scenario where the driver might already be in use

Reply:
Upgrade in use should be, what is making you doubt that?

Reply:
The nvidia-driver script handles that. The other commands just succeed, regardless.